model/registry/gen-ai.yaml

```diff
@@ -20,55 +20,46 @@ groups:
           For example, when using OpenAI client libraries to communicate with Mistral, the `gen_ai.system`
           is set to `openai` based on the instrumentation's best knowledge.
         examples: 'openai'
-        tag: llm-generic-request
       - id: request.model
         stability: experimental
         type: string
         brief: The name of the LLM a request is being made to.
         examples: 'gpt-4'
-        tag: llm-generic-request
       - id: request.max_tokens
         stability: experimental
         type: int
         brief: The maximum number of tokens the LLM generates for a request.
         examples: [100]
-        tag: llm-generic-request
       - id: request.temperature
         stability: experimental
         type: double
         brief: The temperature setting for the LLM request.
         examples: [0.0]
-        tag: llm-generic-request
       - id: request.top_p
         stability: experimental
         type: double
         brief: The top_p sampling setting for the LLM request.
         examples: [1.0]
-        tag: llm-generic-request
       - id: response.id
         stability: experimental
         type: string
         brief: The unique identifier for the completion.
         examples: ['chatcmpl-123']
-        tag: llm-generic-response
       - id: response.model
         stability: experimental
         type: string
         brief: The name of the LLM a response was generated from.
         examples: ['gpt-4-0613']
-        tag: llm-generic-response
-      - id: response.finish_reasons
+      - id: response.finish_reason
```

Comment:
@patcher9 this accidentally bumps you into compatible lol https://github.com/openlit/openlit/issues/318